### PR TITLE
Fixed WormholeVision, Farsee, and calls to filterRooms

### PR DESCRIPTION
**Pull request by PhoenixCodes**
Fixed WormholeVision: Added new trigger line for no visible exits. 
Fixed farsee - local now filters the rooms by current area, and veiled farsee no longer breaks due to old filterRooms call.
Changed all filterRooms calls to the globalized mmp.filterRooms, they were erroring out before.
Added some logic to Worm Sources to remove (house) before doing a room name search, and add it back in the echo.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -5390,7 +5393,7 @@ speedWalkCounter = 0
 mmp.speedWalk = mmp.speedWalk or {}
 mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
-local newversion = "developer"
+local newversion = "20.11.2"
```
Comment (vadi2) [inline mudlet-mapper.xml:5396]: Should still be developer, right?

Comment (PhoenixCodes) [reply mudlet-mapper.xml:5396]: Ahh, this is my first PR here. Didn't know that's how that worked. Fixing!

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -7346,7 +7349,7 @@ end
 function mmp.echonums(roomname, area)
   local t = mmp.searchRoomExact(roomname)
   if area then
-    t = filterRooms(t, area)
+    t = mmp.filterRooms(t, area)
```
Comment (vadi2) [inline mudlet-mapper.xml:7352]: There is a local function at https://github.com/IRE-Mudlet-Mapping/ire-mapping-script/blob/424e11d74fbca728d2fb6146700001027d6c0858/mudlet-mapper.xml#L7333 though, is not working?

Comment (PhoenixCodes) [reply mudlet-mapper.xml:7352]: Hmm. I was getting errors on veiled farsee, prompted me to go through and replace all local calls with the correct global call. The global function is relatively new.

Comment (vadi2) [reply mudlet-mapper.xml:7352]: Ah ok, then the local one should be deleted so we don't get confused

**Comment by PhoenixCodes** [comment]
I'll take care of it tonight and push a new update to this.

Roland Goldberg
----

**Comment by PhoenixCodes** [comment]
Alright, try that. I think it resolves the last comment, I can't find anywhere else where filterRooms was defined globally, but it's definitely a good function to be able to call globally, I use in several scripts of my own.

**Review by vadi2 — APPROVED** [review]
Thanks for the improvements :)

**Pull request merged**